CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 # Changelog
 
-## 6.0.0
+## 6.0.1
```

Comment:
We should go with 6.1.0 as this is not only a bug fix release ;)

Reply:
Whoops you mentioned this yesterday, I was on autopilot this morning :/

Fixed!